tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_go_provisioing_sanity

```diff
@@ -156,7 +156,7 @@ if ( true == via_webhook() && (!(rancher_version ==~ rancher_version_regex)) ) {
                                   string(name: 'GOTEST_TESTCASE', value: "${env.GOTEST_TESTCASE}") ]
 
                       // Rancher server 1 is used to test RKE1, RKE2 and K3s clusters.
```

Comment:
nit - can you remove this comment or not have `Rancher server 1`? The other Jenkinsfile it makes sense as there are multiple Rancher servers.